client/src/api/routes/users.ts

```diff
@@ -89,6 +89,20 @@ export const useDeleteUser = (): UseQueryResult<boolean> => {
     });
 };
 
+/**
+ * @description GET /users/:userId
+ * @param {string} userId 
+ * @returns {UseQueryResult<UserObject>} the requested user
+ * @throws if fails
+ */
+ export const useGetUser = (userId: string): UseQueryResult<UserObject> => {
```

Comment:
we need to get rid of this

Reply:
Yep I deleted these last night but forgot to push but they should be gone now